app/services/reporting/active_users_count_report.rb

```diff
@@ -133,7 +133,7 @@ def cumulative_quarter_ranges
         CalendarService.fiscal_start_date(report_date)..CalendarService.fiscal_q4_start(report_date),
         CalendarService.fiscal_start_date(report_date)..CalendarService.fiscal_end_date(report_date).next_day(1),
       ].map do |range|
-        range.begin.beginning_of_day..range.end.prev_day(1).end_of_day
+        range.begin.beginning_of_day..([range.end.prev_day(1).end_of_day, report_date].min)
```

Comment:
Why is the existing `end_of_day` not enough?   Also, since we’re doing `next_day(1)` and then `prev_day(1)` can we just get rid of both?

Reply:
and maybe to back up, the reason for `prev_day.end_of_day` so that we can have very close time ranges that do not overlap